src/bin/cargo/commands/owner.rs

```diff
@@ -1,51 +1,149 @@
+        // refer to issue: https://github.com/rust-lang/cargo/issues/4352
         .arg(
             multi_opt(
                 "add",
                 "LOGIN",
                 "Name of a user or team to invite as an owner",
             )
-            .short('a'),
+            .short('a')
+            .hide(true),
         )
         .arg(
             multi_opt(
                 "remove",
                 "LOGIN",
                 "Name of a user or team to remove as an owner",
             )
-            .short('r'),
+            .short('r')
+            .hide(true),
         )
-        .arg(flag("list", "List owners of a crate").short('l'))
+        .arg(flag("list", "List owners of a crate").short('l').hide(true))
+        .subcommands([
+            add_arg(
+                Command::new("add")
+                    .about("Name of a user or team to invite as an owner")
+                    .arg_quiet()
+                    .args([
+                        Arg::new("addowner")
+                            .required(true)
+                            .value_delimiter(',')
+                            .value_name("OWNER_NAME")
+                            .help("Name of the owner you want to invite"),
+                        Arg::new("crate")
+                            .value_name("CRATE_NAME")
+                            .help("Crate name that you want to manage the owner"),
+                    ]),
+            ),
+            add_arg(
+                Command::new("remove")
+                    .about("Name of a user or team to remove as an owner")
+                    .arg_quiet()
+                    .args([
+                        Arg::new("removeowner")
+                            .required(true)
+                            .value_delimiter(',')
+                            .value_name("OWNER_NAME")
+                            .help("Name of the owner you want to remove"),
+                        Arg::new("crate")
+                            .value_name("CRATE_NAME")
+                            .help("Crate name that you want to manage the owner"),
+                    ]),
+            ),
+            add_arg(
+                Command::new("list")
+                    .about("List owners of a crate")
+                    .arg_quiet()
+                    .arg(
+                        Arg::new("crate")
+                            .value_name("CRATE_NAME")
+                            .help("Crate name which you want to list all owner names"),
+                    ),
+            ),
+        ])
         .arg(opt("index", "Registry index to modify owners for").value_name("INDEX"))
         .arg(opt("token", "API token to use when authenticating").value_name("TOKEN"))
         .arg(opt("registry", "Registry to use").value_name("REGISTRY"))
         .after_help("Run `cargo help owner` for more detailed information.\n")
 }
 
+fn add_arg(com: Command) -> Command {
```

Comment:
This name should make it clearer what it is doing

Reply:
It has been modified